package.json

```diff
@@ -0,0 +1,29 @@
+{
+  "name": "to-do",
+  "version": "1.0.0",
+  "description": "",
+  "main": ".eslintrc.js",
+  "scripts": {
+    "test": "echo \"Error: no test specified\" && exit 1",
+    "build": "Webpack",
+    "watch": "webpack --watch"
+  },
```

Comment:
- [ ] Kindly add the start script to your `package.json` file.
- [ ] I wasn't able to run the script or start the project as this is not included.